docs/core/diagnostics/sample-debug-target.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Sample debug target
+description: Introduction to the sample debug target used in the diagnostic tutorials.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 12/17/2019
+---
+# Sample debug target
+
+The sample debug target is a simple `webapi` application. The sample triggers undesirable behaviors for the [.NET Core diagnostics tutorials](index.md#net-core-diagnostics-tutorials) to diagnose.
+
+## Sample source
+
+The sample code is in [DiagnosticScenarios](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios). The sample is a basic `webapi` with a single added file [DiagnosticScenarios.cs](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios/Controllers/DiagnosticScenarios.cs).
+
+The simplest way to acquire the source code is to clone the [dotnet/samples](https://github.com/dotnet/sample) repository.
+
+```console
+git clone git@github.com:dotnet/samples.git
```

Comment:
I'd use https instead of SSH. If the reader isn't familiar with using SSH, he might run into some errors.

```suggestion
git clone https://github.com/dotnet/samples.git
```